pkg/types/validation/installconfig.go

```diff
@@ -1064,6 +1064,14 @@ func validateFeatureSet(c *types.InstallConfig) field.ErrorList {
 			if len(c.VSphere.Hosts) > 0 {
 				allErrs = append(allErrs, field.Forbidden(field.NewPath("platform", "vsphere", "hosts"), errMsg))
 			}
+
+			if len(c.AWS.Subnets) == 0 {
```

Comment:
I think this got messed up as a result of the recent merge of https://github.com/openshift/installer/pull/7327

needs to go under an `if c.AWS != nil` conditional. Sorry :(